shipyard_blueprints/sftp/shipyard_sftp/sftp.py

```diff
@@ -18,16 +18,14 @@
 
 logger = ShipyardLogger().get_logger()
 
-DEFAULT_WINDOW_SIZE = 4294967294
-DEFAULT_REKEY_BYTES = pow(2, 40)
-DEFAULT_REKEY_PACKETS = pow(2, 40)
+
 
 
 class SftpClient(CloudStorage):
     EXIT_CODE_DELETE_ERROR = 100
 
     def __init__(
-            self, host: str, port: int, key: str = None, user: str = None, pwd: str = None
+            self, host: str, port: int, key: str = None, user: str = None, pwd: str = None, transport=None
```

Comment:
Not to be overly pedantic, but two points here:
- Whenever something is initialized to None it should be `Optional[type]`
- For the library code in particular, we should have all types annotated. The `transport` variable should be of type `paramiko.Transport`